src/components/Dashboard/Carousel/Carousel.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Slide from './Slide/Slide';
+
+const Carousel = ({ title, description, link }) => {
+  return (
+    <div className="dp-carousel" id="carousel1">
+      <div className="dp-carousel-wrapper dp-carousel-orange">
+        <div className="dp-carousel-content">
+          <Slide></Slide>
```

Comment:
I figure the title, description and link should be props sent to the slide right?
```suggestion
          <Slide title={title} description={description} link={link}></Slide>
```

Reply:
it's right